ivy/functional/backends/jax/experimental/losses.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
+# coding=utf-8
+# Copyright 2022 The HuggingFace Team All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Fine-tuning the Flax library models for connectionist temporal classification (CTC) speech recognition.
+"""
+# You can also adapt this script on your own sequence to sequence task. Pointers for this are left as comments.
+
+import math
+from typing import Optional, Tuple, Sequence, Union
+
+import jax.nn
+import jax.numpy as jnp
+from jax import jit
+
+from ivy.functional.backends.jax import JaxArray
```

Comment:
import this from jax itself not ivy jax

Reply:
when I look at other codes I saw people imported like this and when I hover over JaxArray pycharm shows me it belongs to ivy.functional.backend.jax

Reply:
do you want me to use jax.numpy.array?

Reply:
andI can't import  JAxArray from jax directly.

Reply:
jax.numpy.ndarray
I am gonna try this